crates/bevy_pbr/src/material.rs

```diff
@@ -202,11 +223,15 @@ pub struct MaterialPipeline<M: SpecializedMaterial> {
     marker: PhantomData<M>,
 }
 
-impl<M: SpecializedMaterial> SpecializedPipeline for MaterialPipeline<M> {
+impl<M: SpecializedMaterial> SpecializedMeshPipeline for MaterialPipeline<M> {
     type Key = (MeshPipelineKey, M::Key);
```

Comment:
I don't like the `.0` and `.1` here; it makes this code much harder to follow.

This is a pre-existing issue with `MaterialPipeline`; if you're opposed to bloating this PR I can make a new issue.

Reply:
Good call. Fixed!